apis/v1beta1/httproute_types.go

```diff
@@ -198,8 +198,14 @@ type HTTPRouteRule struct {
 	// All filters are expected to be compatible with each other except for the
 	// URLRewrite and RequestRedirect filters, which may not be combined. If an
 	// implementation can not support other combinations of filters, they must clearly
-	// document that limitation. In all cases where incompatible or unsupported
-	// filters are specified, implementations MUST add a warning condition to status.
+	// document that limitation. A Route containing a single rule with incompatible
+	// filters must result in the implementation setting the Accepted Condition for
+	// the Route to `status: False`, with a Reason of `UnsupportedValue`. A Route
```

Comment:
It could be useful to give an example of what an incompatible filter might look like. Is this referring to unsupported filters, unsupported combinations of filters, or something else entirely?

In any case, the condition and reason we require here should line up with what we decide below. I think I'd personally prefer just adding a new "IncompatibleFilters" reason.

As one last question, are there any combination of filters that we know will be incompatible for everyone? Would combining response header modification and redirects ever be possible and/or desired?

Reply:
> Is this referring to unsupported filters, unsupported combinations of filters, or something else entirely

this is mainly in response to the fact that URLRewrite and RequestRedirect filters are mentioned above and in GEP-1364 as incompatible

Reply:
> Would combining response header modification and redirects ever be possible and/or desired?

Yeah I'm not sure, seems possible a controller might have some use for this

Worth adding a conformance test for this combination so we capture feedback

Reply:
I think a reason more like `FilterConflict` may be more clear than the nuance between "Unsupported" and "Incompatible", and has better parity with existing similar reasons like `HostnameConflict` and `ProtocolConflict` https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io%2fv1beta1.ListenerConditionType